Machina Player

McBen opened the pull request:
Machina is now included in chat log.
This leads to some minor problems like chat format and player icons in player-tracker


This PR 
a) adds the neutral chat names
![grafik](https://github.com/IITC-CE/ingress-intel-total-conversion/assets/29117/b74ff41e-ddab-46a4-8910-86230b952507)
white because it's reported as "neutral" and not as MAC.

b) ignore neutral players in player-tracker plugin

McBen committed:
Chat: Format neutrals (machina)

McBen committed:
PlayerTracker: Ignore neutral players (Machina)

xscreach commented on plugins/player-activity-tracker.js:
```diff
@@ -169,7 +169,7 @@ window.plugin.playerTracker.processNewData = function(data) {
 
     // skip unusable events
-    if(!plrname || !lat || !lng || !id || skipThisMessage) return true;
+    if (!plrname || !lat || !lng || !id || skipThisMessage || json[2].plext.team === 'NEUTRAL') return true;
```
Rather use `teamStringToId(...team) === window.TEAM_NONE`

McBen committed:
plugins: Add machina tracker

this is basically a copy of the player tracker. But it will track only neutral player and
mark all their locations instead of drawing a path

McBen commented:
7219019 added a new plugin for machina tracking (show markers instead of the path)

xscreach commented on plugins/machina_tracker.user.js:
```diff
@@ -0,0 +1,263 @@
+    });
+
+    // skip unusable events
+    if (!plrname || !lat || !lng || !id || skipThisMessage || json[2].plext.team !== 'NEUTRAL') return true;
```
Rather use `teamStringToId(...team) !== window.TEAM_NONE`

McBen commented:
Example of the new plugin:
![grafik](https://github.com/IITC-CE/ingress-intel-total-conversion/assets/29117/315868c2-bb03-469e-9be1-63806d0224de)

McBen committed:
Chat: use teamStringToId

McBen committed:
chat: (codestyle) making eslint happy

xscreach commented on core/code/chat.js:
```diff
@@ -1,3 +1,5 @@
+/* global TEAM_TO_CSS */
```
These are defined under window. We should imho rather use e.g. `window.teamToStringId` than keep defining these in files again

xscreach reviewed (changes requested):
Small clean up required, see commented lines
